src/main/java/frc/robot/subsystems/drive/CommandSwerveDrivetrain.java

```diff
@@ -20,14 +20,13 @@
  */
 public class CommandSwerveDrivetrain extends SwerveDrivetrain implements Subsystem {
```

Comment:
If we do what they do in the examples, it is way better: https://github.com/CrossTheRoadElec/Phoenix6-Examples/blob/main/java/SwerveWithPathPlanner/src/main/java/frc/robot/CommandSwerveDrivetrain.java

They basically make another thread and run the simulator faster

Reply:
Sounds good, I'll add their implementation